sync2/handler2/handler.go

```diff
@@ -493,8 +500,14 @@ func (h *Handler) EnsurePolling(p *pubsub.V3EnsurePolling) {
 
 func typingHash(ephEvent json.RawMessage) uint64 {
 	h := fnv.New64a()
-	for _, userID := range gjson.ParseBytes(ephEvent).Get("content.user_ids").Array() {
-		h.Write([]byte(userID.Str))
+	parsedUserIDs := gjson.ParseBytes(ephEvent).Get("content.user_ids").Array()
```

Comment:
Sorting the userIDs should, in theory, not be needed, but ensures that we don't compare hashes for "alice, bob" with "bob, alice" which would otherwise be different.